projects/m2/004-avoiding-duplicates/js/index.js

```diff
@@ -0,0 +1,33 @@
+const prompt = require("prompt-sync")();
+let wordsArray = [];
```

Comment:
Remember that in this case you can use `const` ;-)
https://www.w3schools.com/js/js_array_const.asp